components/raftstore/src/store/util.rs

```diff
@@ -912,7 +912,7 @@ pub fn get_entry_header(entry: &Entry) -> RaftRequestHeader {
 pub fn parse_data_at<T: Message + Default>(data: &[u8], index: u64) -> T {
     let mut result = T::default();
     result.merge_from_bytes(data).unwrap_or_else(|e| {
-        panic!("{} data is corrupted : {:?}", index, e);
+        panic!("{} data is corrupted : {:?}. hex value: {}", index, e, log_wrappers::Value::value(data));
```

Comment:
region id is important.

Reply:
The tag is removed by #14335, so shall we add it back?

Reply:
Yes. And you can add region_id and peer_id as parameters and log them like `[region_id={}]`.